ar/accessibility-in-app.md

```diff
@@ -7,7 +7,7 @@ title: Accessibility
 # Accessibility
 
 <p class="md-block-lead md-text-color-RO-donkerblauw" markdown="1">
-  The CoronaMelder app should be accessible to everyone. From its very first design accessibility has been taken into account, with the help of experts and experience experts. Yet the app might not be as accessible to you. Are you unable to use the app well? Let us know by emailing <a href="mailto:coronamelder@minvws.nl">coronamelder@minvws.nl</a>.
+  The CoronaMelder app should be accessible to everyone. From its very first design accessibility has been taken into account, with the help of experts and experience experts. Yet the app might not be as accessible to you. Are you unable to use the app well? Let us know by emailing <a href="mailto:coronamelder@minvws.nl?subject={{ site.data.translations.accessibility-in-app-email-subject[page.lang] }}" target="_blank">.
```

Comment:
I think this would be a great change, (though I do have to talk to the content manager of the inline html isn't estranging them from editing the content too much), but I notice these links don't close with a link text:
```
<a href="mailto...." target="_blank">
```
I think you mean
```
<a href="mailto...." target="_blank" rel="noreferrer">coronamelder@minvws.nl</a>
```
(I have added a `rel="noreferrer"` attribute here because I've learned it's best practice since `target="_blank"` is known to cause security issues, even though I have to admit I am not sure if that counts for mailto: links).
https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/